electrum/tests/test_readability.py

```diff
@@ -0,0 +1,22 @@
+from decimal import Decimal
+
+from electrum.util import (format_satoshis, format_fee_satoshis, parse_URI,
+                           is_hash256_str, chunks, is_ip_address, list_enabled_bits,
+                           format_satoshis_plain, is_private_netaddress, is_hex_str,
+                           is_integer, is_non_negative_integer, is_int_or_float,
+                           is_non_negative_int_or_float)
+
+from . import ElectrumTestCase
+
+
+class Test_Satoshi_Readability(ElectrumTestCase):
+
+    def test_format_satoshis_readability(self):

```

Comment:
Please move this test to `test_util.py/TestUtil`, next to the existing unit tests that exercise `format_satoshis`.
No need to create a new file just for this.

Reply:
For the name of the function, I suggest: `test_format_satoshis_add_thousands_sep` instead.